[2023-12-07 01:31:18,735 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:31:35,597 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:37:00,545 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:37:21,040 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:37:22,834 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:38:21,803 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:38:23,193 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:38:57,429 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 71, in generate_excel_report_data
    cell.column_dimensions = cell.column_dimensions + 2  # Adjust width for padding
AttributeError: 'Cell' object has no attribute 'column_dimensions'
[2023-12-07 01:41:14,014 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:41:35,871 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 71, in generate_excel_report_data
    cell.column_dimensions = 15  # Adjust width for padding
AttributeError: 'Cell' object has no attribute 'column_dimensions'
[2023-12-07 01:42:28,756 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:42:37,985 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 79, in generate_excel_report_data
    cell.row_dimensions = cell.row_dimensions * cell_wrap_count  # Adjust row height
AttributeError: 'Cell' object has no attribute 'row_dimensions'
[2023-12-07 01:42:50,852 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:42:55,093 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 84, in generate_excel_report_data
    workbook.save(report_path)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/workbook/workbook.py", line 386, in save
    save_workbook(self, filename)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 294, in save_workbook
    writer.save()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 275, in save
    self.write_data()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 77, in write_data
    self._write_worksheets()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 215, in _write_worksheets
    self.write_worksheet(ws)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 200, in write_worksheet
    writer.write()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 358, in write
    self.write_top()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 102, in write_top
    self.write_format()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 75, in write_format
    self.ws.sheet_format.outlineLevelCol = self.ws.column_dimensions.max_outline
AttributeError: 'int' object has no attribute 'max_outline'
[2023-12-07 01:44:18,765 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:44:23,608 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 90, in generate_excel_report_data
    workbook.save(report_path)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/workbook/workbook.py", line 386, in save
    save_workbook(self, filename)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 294, in save_workbook
    writer.save()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 275, in save
    self.write_data()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 77, in write_data
    self._write_worksheets()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 215, in _write_worksheets
    self.write_worksheet(ws)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/writer/excel.py", line 200, in write_worksheet
    writer.write()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 358, in write
    self.write_top()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 102, in write_top
    self.write_format()
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/worksheet/_writer.py", line 75, in write_format
    self.ws.sheet_format.outlineLevelCol = self.ws.column_dimensions.max_outline
AttributeError: 'int' object has no attribute 'max_outline'
[2023-12-07 01:44:53,062 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:45:59,478 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:48:40,025 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:51:18,361 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:53:23,525 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:54:40,419 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 01:55:16,464 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 63, in generate_excel_report_data
    cell.alignment.horizontal = 'justify'  # Or use other options like 'center' or 'left'
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/styles/proxy.py", line 29, in __setattr__
    raise AttributeError("Style objects are immutable and cannot be changed."
AttributeError: Style objects are immutable and cannot be changed.Reassign the style with a copy
[2023-12-07 01:56:33,971 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 01:57:37,313 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:02:19,638 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:02:54,203 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:04:09,406 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:04:40,429 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:06:01,461 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py changed, reloading.
[2023-12-07 02:07:23,017 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:07:35,514 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:09:01,078 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py changed, reloading.
[2023-12-07 02:10:33,007 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:15:19,194 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py changed, reloading.
[2023-12-07 02:15:54,064 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 300, in process_compiler_report
    self._generate_driverwise_count_sheet_data()
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 35, in _generate_driverwise_count_sheet_data
    df_driverwise_count[self.filewise_count_header[1]] = pd.Categorical(df_driverwise_count[self.filewise_count_header[1]], categories=drivers.tolist(), ordered=True)
AttributeError: 'list' object has no attribute 'tolist'
[2023-12-07 02:16:33,752 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py changed, reloading.
[2023-12-07 02:17:01,650 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:19:27,581 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:19:32,912 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:19:36,294 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:20:24,131 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:21:07,687 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:22:18,568 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:23:26,206 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:24:07,086 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:24:20,147 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:24:57,055 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:25:34,047 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:25:54,635 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:26:28,939 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:28:20,995 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:28:38,885 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:29:23,238 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:31:53,824 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:31:58,106 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:32:34,739 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:34:11,756 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:34:15,679 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:34:55,381 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:36:41,744 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:36:54,061 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:37:30,265 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:38:26,022 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:38:29,300 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:39:04,092 GMT] ERROR::log_response() Internal Server Error: /process_log_warnings/
Traceback (most recent call last):
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_gui/views.py", line 26, in process_log_warnings
    report_data, report_name = log_dashboard.process_compiler_warnings(files_data)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/app_dashboard.py", line 66, in process_compiler_warnings
    report_data = cwr().process_compiler_report(log_directory=log_and_report_dir_path, report_path=report_path, props=self.props)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/services/compiler_warning_ops.py", line 303, in process_compiler_report
    self.report_data = reports.generate_excel_report_data(report_path, self.df_data_dict)
  File "/ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py", line 66, in generate_excel_report_data
    cell.alignment = Alignment(horizontal='center', vertical='middle')  # Center align the content
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/styles/alignment.py", line 50, in __init__
    self.vertical = vertical
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/descriptors/base.py", line 147, in __set__
    super(NoneSet, self).__set__(instance, value)
  File "/home/saldanha/miniconda3/envs/log_analysis/lib/python3.8/site-packages/openpyxl/descriptors/base.py", line 132, in __set__
    raise ValueError(self.__doc__)
ValueError: Value must be one of {'justify', 'distributed', 'center', 'bottom', 'top'}
[2023-12-07 02:39:15,111 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:39:45,894 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:41:18,791 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:41:24,880 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:42:03,348 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:44:50,337 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:44:56,758 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:45:32,332 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:48:37,952 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:48:49,244 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:49:26,983 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:51:12,866 GMT] INFO::trigger_reload() /ubuntuwork/LearnProjectsVSCode/poc_log_analysis/log_analysis_api/src/utils/report_utils.py changed, reloading.
[2023-12-07 02:51:26,341 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:52:03,277 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:52:56,746 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 02:53:30,926 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:12:53,846 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:16:16,466 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:17:07,896 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:17:36,405 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:17:41,644 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:17:43,820 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:18:31,502 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:19:21,854 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:19:43,734 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:29:50,422 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:34:03,525 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:34:58,744 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:35:16,218 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:35:23,574 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:35:26,533 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:36:21,650 GMT] WARNING::log_response() Not Found: /favicon.ico
[2023-12-07 03:36:29,884 GMT] WARNING::log_response() Not Found: /favicon.ico
